src/Chapter1.hs

```diff
@@ -411,7 +416,7 @@ task is to specify the type of this function.
 >>> squareSum 3 4
 49
 -}
-
+squareSum :: Num a => a -> a -> a
```

Comment:
Wow, you are already one step ahead! We haven't explained polymorphism yet, but it looks like you get it. Nice one :top: 
For this task it would be enough for us to see `squareSum :: Int -> Int -> Int`, but you don't need to change back, as your type signature is totally valid and even more general :+1: